pkgs/top-level/all-packages.nix

```diff
@@ -5581,6 +5581,21 @@ with pkgs;
 
   gcc_multi = wrapCCMulti gcc;
 
+  llvm-multi = llvmPkgs:
+    if system == "x86_64-linux" then
+      callPackages ../development/compilers/llvm/multi.nix {
+        gcc32 = pkgsi686Linux.gcc;
+        gcc64 = pkgs.gcc;
+        llvmPackages = llvmPkgs;
```

Comment:
The only thing that `multi.nix` uses from `llvmPackages` is `clang`, so it's better to pass `clang` rather than the whole of `llvmPackages`.